qa/automation/Starynina.Scenarios/3. Send CV/3.3.Send CV.feature

```diff
@@ -0,0 +1,134 @@
+Feature: 3.3. Send CV
+
+Background:
+Given user is on the page with description of QA Engineer (Life Science project) position https://career.quantori.com/ru/positions/qa-engineer-life-science-project
+
+Scenario: 3.3_01 User can use the "Send CV" button
+	When user clicks the "Send CV" button
+	Then the page is being scrolled down to the CV form
+
+Scenario: 3.3_02 User can scroll the page
+	When user uses the scroll on the side bar
+	Then the page is being scrolled down to the CV form
+	
+Scenario: 3.3_03 User can't send empty CV form
+	When user clicks the "Send" button
+	Then the error messages "This field is required" for "Name" and "Email" fields is appeared
+	And the "Name" and "Email" field, "Attach CV" button and the checkbox "I agree to the personal data processing" are highligted in red
+	And the CV form is not sent
+
+Scenario: 3.3_04 User can choose CV file from local PC
+	When user clicks the "Attach CV" button
+	Then the window to choose the file from local PC is opened
+
+Scenario Outline: 3.3_05 User enters various types of name in the CV form 
+	When user enters <name> in the "Name" field
+	Then the result <result> is appeared
+
+	Examples:
+	| name                    | result                   |
+	|                         | "This field is required" |
+	| n                       | "Invalid name format"    |
+	| nina                    | "Invalid name format"    |
+	| nina 1988               | "Invalid name format"    |
+	| /*-+?:%; 909            | "Invalid name format"    |
+	| n s                     | accepted                 | 
+	| nina star               | accepted                 | 
+	| (two words in Cyrillic) | accepted                 | 
+
+Scenario: 3.3_06 User enters name with more than maximum characters in the CV form 
+	When user enters name of 71 characters or more in the "Name" field
+	Then the error messages "The maximum number of characters is 70" for "Name" field is appeared
+	And the "Name" field is highligted in red
+
+Scenario Outline: 3.3_07 User enters various types of email in the CV form
+	When user enters <email> in the "Email" field
+	Then the result <result> is appeared
+
+	Examples:
+	| email                      | result                   |
+	|                            | "This field is required" |
+	| nina.gmail.com             | "Invalid name format"    |
+	| 123                        | "Invalid name format"    |
+	| nina@mail@.com             | "Invalid name format"    |
+	| star@gmail.com             | accepted                 | 
+	| star@mail.ru               | accepted                 | 
+	| "wordInCyrillic"@gmail.com | accepted                 | 
```

Comment:
everything must be spelled out, without abstract examples, so that it can be automated